lib/transforms/json-to-html-data.js

```diff
@@ -220,9 +220,6 @@ function getMaterials (data) {
   var values;
   if (data.attributes.materials) {
     values = data.attributes.materials.map((material, i, arr) => {
```

Comment:
I think there is the same thing a bit under in the code with ```data.attributes.component.forEach((el, i, arr) => {``` I don't think we use i and arr so we can clean them too?